python/pyspark/sql/connect/logging.py

```diff
@@ -21,30 +21,35 @@
 import os
 from typing import Optional
 
-__all__ = [
-    "getLogLevel",
-]
+__all__ = ["configure_logging", "getLogLevel"]
 
 
-def _configure_logging() -> logging.Logger:
-    """Configure logging for the Spark Connect clients."""
+def configure_logging(level: Optional[str] = None) -> logging.Logger:
```

Comment:
Should we rename this to `cofgirueLogging` since it's an API now?

Reply:
Good call.